File: module_main/src/main/java/com/phone/module_main/model/IMainModel.java
package com.phone.module_main.model;

import java.util.Map;

import io.reactivex.Observable;
import okhttp3.ResponseBody;

/**
 * author    : Urasaki
 * e-mail    : 1164688204@qq.com
 * date      : 2019/3/10 16:57
 * introduce :
 */

public interface IMainModel {

    Observable<ResponseBody> mainData(Map<String, String> bodyParams);
}

File: module_main/src/main/java/com/phone/module_main/model/IMainModel.kt
package com.phone.module_main.model

import io.reactivex.Observable
import okhttp3.ResponseBody

/**
 * author    : Urasaki
 * e-mail    : 1164688204@qq.com
 * date      : 2020/3/10 17:03
 * introduce :
 */

interface IMainModel {

    fun mainData(bodyParams: Map<String, String>): Observable<ResponseBody>
}